src/Samples/PortableTask/PortableTask.csproj

```diff
@@ -12,7 +12,10 @@
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="Microsoft.Build.Framework" Version="15.5.180" />
-    <PackageReference Include="Microsoft.Build.Utilities.Core" Version="15.5.180" />
+    <PackageReference Include="Microsoft.Build.Utilities.Core" Version="15.5.180" GeneratePathProperty="true" />
   </ItemGroup>
   <Target Name="UpdateXlf" />
+  <Target Name="CopyMSBuildUtilitiesToTemp" AfterTargets="Build">
```

Comment:
Doesn't copy to temp any more. Also `AfterTargets="Build"` is bad style; can you be more specific?

Reply:
It really just has to happen at any point after the PackageReference is resolved and before tests start running...after the build is otherwise finished is one such time. I guess I could try to put it after ResolvePackageSomething, but I'd have to figure out what exactly that is.